NEWS.md

```diff
@@ -1,3 +1,25 @@
+## scoringutils 0.1.3
+
+### (Potentially) Breaking changes
```

Comment:
This is a good note. I think making multiple breaking changes at once is probably the best idea. And you are right changing to singular names is the most consistent.